### ISSUE TITLE
 `num_features` of `nn.BatchNorm1d()` with a tuple of `int` works

### ISSUE THREAD

**Issue by hyperkai** [pytorch/pytorch#136138]
### 🐛 Describe the bug

[The doc](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm1d.html) of `nn.BatchNorm1d()` says `num_features` is `int` as shown below:

> Parameters
> - num_features ([int](https://docs.python.org/3/library/functions.html#int)) – number of features or channels C of the input

But `num_features` with a tuple of `int` works as shown below. *It also happens with `num_features` of [nn.BatchNorm2d()](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm2d.html) and [nn.BatchNorm3d()](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm3d.html):

```python
import torch
from torch import nn

my_tensor = torch.tensor([[8., -3., 0.],
                          [1., 5., -2.]])
batchnorm1d = nn.BatchNorm1d(num_features=(3,))
batchnorm1d(input=my_tensor)            # ↑↑↑↑
# tensor([[1.0000, -1.0000, 1.0000],
#         [-1.0000, 1.0000, -1.0000]], grad_fn=<NativeBatchNormBackward0>)
```

### Versions

```python
import torch

torch.__version__ # '2.3.0'
```
